src/library/scala/collection/immutable/Range.scala

```diff
@@ -518,4 +518,11 @@ private class RangeIterator(
     _next = value + step
     value
   }
+
+  override def drop(n: Int): Iterator[Int] = {
+    if (step > 0) _next = Math.min(lastElement, _next + step * n)
+    else if (step < 0) _next = Math.max(lastElement, _next + step * n)
```

Comment:
`_next + step * n` needs to be guarded against integer overflows in these expressions

Reply:
will do

Reply:
done